### PR TITLE
New Feature Calculator UI

### PR DESCRIPTION
**Pull request by miguelesco**
## What's new

- Remove the previous code generate by default.
- Add Calculator class.
- Add Keyboard Class with his childrens
- Styles generals and components render with styled-component

## Live Preview 

https://miguelesco.github.io/math-magicians

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+function App() {
+  return (
+    <react.Fragment>
+      <GlobalStyles />
+      <Calculator />
+    </react.Fragment>
+  );
+}
+
+export default App;
```
Comment (DeliceLydia) [inline src/App.js]: - [x] `app.js` component should be a class please change that

File: src/components/simbols_keyboard/simbols.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable linebreak-style */
+import React from 'react';
+import Container, { SimbolsBtn } from './styles';
+
+const symbolsArray = ['+', 'x', '-', '+', '='];
+
```
Comment (DeliceLydia) [inline src/components/simbols_keyboard/simbols.jsx]: - [x] The first operator is `division` not `+` please fix that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @miguelesco, 

Good job so far! there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

- [x] Please remove all the boilerplate in the `public` directory and remain with `index.html` only because they are not useful for now and `App.test.js`
- [x] In the screen of your  calculator instead of having a default `8` change it to `zero` to follow the design


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @miguelesco 😃, 

Great job implementing the requested changes. 💪 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**